modules/project_cleanup/function_source/main.go

```diff
@@ -224,6 +243,46 @@ func getCleanUpTagKeysOrTerminateExecution() bool {
 	return result
 }
 
+func getFeedsListFromEnv(envVariableName string) []*regexp.Regexp {
+	targetIncludedFeeds := os.Getenv(envVariableName)
+	logger.Println("Try to get CAI Feeds list")
+	if targetIncludedFeeds == "" {
+		logger.Printf("No CAI Feeds provided.")
+		return nil
+	}
+
+	var caiFeeds []string
+	err := json.Unmarshal([]byte(targetIncludedFeeds), &caiFeeds)
+	if err != nil {
+		logger.Printf("Failed to get CAI Feeds list from [%s] env variable, error [%s]", envVariableName, err.Error())
+		return nil
+	}
+
+	regexFeeds := make([]*regexp.Regexp, len(caiFeeds))
+	for i, feed := range caiFeeds {
+		regexFeeds[i], err = regexp.Compile(feed)
+		if err != nil {
+			logger.Printf("Failed to compile regex for CAI Feed [%s], error [%s]", feed, err.Error())
+			regexFeeds[i] = regexp.MustCompile(".*")
```

Comment:
Could we skip adding to `regexFeeds[i]` if err?

Reply:
fixed.